bigbluebutton-html5/imports/ui/components/session-details/component.tsx

```
@@ -0,0 +1,213 @@
+        isOpen,
+        onRequestClose,
+        priority,
+        anchorElement,
+      }}
+    >
+      <Styled.Chevron />
+      <Styled.Container
+        isFullWidth={isMobile || !(loginUrl || (formattedDialNum && formattedTelVoice))}
+      >
+        <div>
+          <Styled.WelcomeMessage dangerouslySetInnerHTML={{ __html: welcomeMessage }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)